utils/src/main/kotlin/OkHttpClientHelper.kt

```diff
@@ -71,10 +71,13 @@ object OkHttpClientHelper {
         // proxy authenticator.
         return OkHttpClient.Builder()
             .addNetworkInterceptor { chain ->
+                val request = chain.request()
                 chain.proceed(
-                    chain.request().newBuilder()
-                        .header("User-Agent", Environment.ORT_USER_AGENT)
-                        .build()
+                    if (request.header("User-Agent") == null) {
```

Comment:
Proposal: The logic to add a user agent header if it is missing could be moved to a helper function, e.g. _addAgentHeaderIfMissing()_. Then adding the interceptor becomes shorter and easier to read:
`.addNetworkInterceptor { chain -> chain.proceed(addAgentHeaderIfMissing(chain.request()) }`

Reply:
I don't think it's worth it for this single use case.